hooks/configure

```diff
@@ -74,22 +74,29 @@ switch_service() {
 	esac
 }
 
-update_pi_config_value() {
-    path="$1"
-    key="$2"
-    value="$3"
+# helper that checks if the given tmp_path can be written
+is_writable() {
+    tmp_path="$1"
+    printf "" 2>/dev/null >> "$tmp_path"
```

Comment:
The name doesn't reflect the fact that the resulting file is created, if missing.